tests/test_extract.py

```diff
@@ -1,176 +1,179 @@
-                tweak_info = chip.meta.get('tweakwcs_info', None)
-                chip_id = chip.meta.get('chip_id',1)
-                # determine 30mas limit in pixels
-                xylimit = 30
-                # Perform comparisons
-                nmatches = tweak_info['nmatches']
-                xrms,yrms = tweak_info['rms']
-                xrms *= chip.wcs.pscale*1000
-                yrms *= chip.wcs.pscale*1000
-                if any([nmatches==0, xrms>xylimit, yrms>xylimit]):
-                    failures = True
-                    msg1 = "Observation {}[{}] failed alignment with "
-                    msg2 = "    RMS=({:.4f},{:.4f})mas [limit:{:.4f}mas] and NMATCHES={}"
-                    print(msg1.format(infile, chip_id))
-                    print(msg2.format(xrms, yrms, xylimit, nmatches))
-            
-        assert(not failures)
-    
+import sys
+import traceback
+import os
+import pytest
+from astropy.table import Table
+from astropy.io import fits
+
+import tweakwcs
+from base_test import BaseHLATest
+import hlapipeline.utils.astrometric_utils as amutils
+from hlapipeline.alignimages import generate_source_catalogs
+from ci_watson.artifactory_helpers import get_bigdata
+
+
+class TestPipeline(BaseHLATest):
+    """ Tests which validate whether pipeline code will meet requirements.
+
+        Characeteristics of these tests:
+          * A single astrometric catalog was obtained with both GAIA and non-GAIA
+            (PanSTARRS?) sources for the entire combined field-of-view using the GSSS
+            server.
+          * This test only determines whether enough sources
+            extracted from the images can match the objects included in
+            the astrometric catalog.
+          * Tests are included to cover most observation modes
+            (ACS/SBC, WFC3/IR,...)
+          * The WCS information for the input exposures do not get updated in this test.
+          * No mosaic gets generated.
+
+            * For observations with >3 astrometric sources, at least 3 sources
+            * For observations with 3 or fewer sources, all astrometric sources
+              were identified in the image
+
+        The following datasets are used in these tests:
+
+            * j8ep04lwq : ACS/SBC dataset with >10 sources (in image and in GAIADR2)
+            * J8D806010 : ACS/WFC dataset dominated by CRs and 15 GAIA sources
+            * IB2V09010 : WFC3/IR dataset with 24 GAIA sources in field-of-view
+    """
+
+    ref_loc = ['truth']
+
+    @pytest.mark.parametrize("input_filenames, truth_file",
+                                [('j8ep04lwq','j8ep04lwq_sky.cat'),
+                                 ('J8D806010','j8d806bgq_sky_cat.ecsv'),
+                                 ('IB2V09010', 'ib2v09kzq_sky_cat.ecsv')]
+                            )
+    def XXtest_generate_catalog(self,input_filenames, truth_file):
+        """ Verify whether sources from astrometric catalogs can be extracted from images.

```

Comment:
I will assume this test has been deliberately disabled at this time.

Reply:
Actually, this was an oversight from my own runs of these tests.  It has been turned back on in https://github.com/spacetelescope/hlapipeline/pull/43/commits/a1f683ca6072d6466d06009e1b2884cce548cd5e